matcalc/phonon.py

```diff
@@ -67,9 +67,9 @@ def calc(self, structure: Structure) -> dict:
             thermal_properties:
                 {
                     temperatures: list of temperatures in Kelvin,
-                    free_energy: list of Helmholtz free energies at corresponding temperatures in eV,
-                    entropy: list of entropies at corresponding temperatures in eV/K,
-                    heat_capacity: list of heat capacities at constant volume at corresponding temperatures in eV/K^2,
+                    free_energy: list of Helmholtz free energies at corresponding temperatures in KJ/mol,
```

Comment:
thanks! can you add a code comment with a link to where the units are documented?

Reply:
How about just add the comments and links in this PR? Or it may affect the aesthetics of the source code.

Reply:
feel free to insert the link where you think best